### ISSUE TITLE
Suggestion to change address schema

### ISSUE THREAD

**Issue by Jaikant** [beckn/protocol-specifications#213]
The current [address schema](https://developers.becknprotocol.io/docs/logistics-specification/schema-reference/address/) has the following fields like `locality`, `ward`, `street`, `building`, `door` in addition to other ones.

I feel the above fields may be problematic to deal with - due to inconsistencies in the way people write addresses.

An easier and more flexible approach would be to have an `array` of `address_lines` in place of the above. Keeping the rest of the fields like `name`, `city`, `state`, `country` & `area_code`

**Comment by pkapustin** [comment]
Agree completely, we already did this in this in the parking network spec with @techframewirk.

**Comment by core-wg-admin** [comment]
Good point. Address can be made as an array of address lines of type string.

@venkatramanm @pramodkvarma

**Comment by pramodkvarma** [comment]
I assume this is only for unstructured address fields. Structured fields like `state`, `postal code`, and `geo location` need to be explicitly captured. I am ok to have an array for unstructured part of an address. 

Please also stress test using few global addresses.

**Comment by emmayank** [comment]
The `Address` schema has been reduced to a simple string type. Implementing networks can specify address format depending upon local context as part of network policy.

Furthermore @pramodkvarma `state`, `area_code`, and `gps` already exist alongside `address` in the `Location` schema.

Closing this issue, as it seems to have been resolved.
